jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseServiceImplTest.java

```diff
@@ -1916,4 +1917,134 @@ public void testTriggerNotExistingAdHocFragment() {
         }
     }
 
+    @Test
+    public void testStartEmptyCaseUsingLoggedInOwner() {
+        assertNotNull(deploymentService);
+        DeploymentUnit deploymentUnit = new KModuleDeploymentUnit(GROUP_ID, ARTIFACT_ID, VERSION);
+
+        deploymentService.deploy(deploymentUnit);
+        units.add(deploymentUnit);
```

Comment:
Adding a deployment unit is a step of preparing a test environment, thus is should be called from a `@Before` method - that will reduce duplication, as every test method in this PR starts with this block of code.